.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+*.sh text eol=lf
```

Comment:
Proc je tam .sh zvlast? Celkove bych preferoval, abys to vyresil u sebe v editoru (mas PyCharm?), nebo, idealne, nepouzival windows :)

Reply:
Tak protože u ničeho jiného to nevadí, Python si vesele pobere obojí, ale CRLF .sh skript fakt nepustíš. Ale jako já to tam nepotřebuji, ale když si to někdo stáhne na windows s autocrlf tak tohle mu dovolí ten skript v dockeru nebo wsl pořád spustit

Reply:
kdyz nekdo zmeni konce radku na crlf, tak mu to nikdy nemergnu, fuj

Reply:
asi ten soubor hod uplne do pryc pls

Reply:
Tak on to git nebude měnit sem, jen na lokálu, proto jsem si motal i v tom sám, protože git to dělá by default, ale když si to stáhne k sobě tak tohle mu pomůže to spustit (i když ani mě ani tebe se to netýká tak to tu může být)